ui.js

```diff
@@ -0,0 +1,67 @@
+// Called after a WIN or a LOSS.
+// Intended to disconnect the user interface from the game state, 
+// freezing the html and preventing the user from playing the game
+// any further.
+// User would be forced to either quit or refresh the page for
+// a new game.
+var stopGame = function () {
+	// $("script:eq(1)").remove();	//#1
+
+	//$("#USER_INTERFACE").remove();	//#2
+	
+	//$(document).replace("script", "NOT_A_SCRIPT");	//#3
+
+	// var replaceLoc = $(document.head);	//#4
+	// var regExMatch = "script";
+	// var replaceWith = "NOT_A_SCRIPT";
+	// var result = replaceLoc.replace(regExMatch, replaceWith);
+	// $("head").html(result);
+};
+
+
+$(document).ready(function() {
+	document.oncontextmenu = function() {return false;};
+	var grid = new Grid(5,5,2);
+	var report = grid.print();
+	
+	$('.cell').mousedown(function(event) {
+		var row = $(this).data('row');
+		var col = $(this).data('col');
+		//right-click event for flags
+		if(event.which === 3) {
+			if(grid.setFlag(row,col)) {
```

Comment:
So I tried this locally in your code and it doesn't work because the event is still `mousedown`. `e.preventDefault()` on mousedown won't stop the right click `contextmenu` event from happening.

You will still need to bind a `contextmenu` event handler for each cell.